### PR TITLE
JS Task #1

### PR DESCRIPTION
**Pull request by dreyhiflden**
Task 1.1 is done, but 1.2 still in progress. This is not final version for now.

### CODE REVIEW

File: to_hex.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+let toHex = value => {
+
+  let hexVal = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "A", "B", "C", "D", "E", "F"],
```
Comment (carrot-e) [inline to_hex.js]: Use single quotes (by code convention).

File: to_hex.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      reminder,
```
Comment (carrot-e) [inline to_hex.js]: remainder*

File: index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>JS task 1</title>
+</head>
+<body>
+
+<script src="sum_odd_numbs.js.js"></script>
```
Comment (carrot-e) [inline index.html]: `.js.js`

File: to_hex.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      returnValue = '';
+
+  while (value !== 0) {
+    reminder = value % 16;
+    value = parseInt((value / 16), 10);
+    returnValue = hexVal[reminder] + returnValue;
+  }
+
+  return `${returnValue}`;
```
Comment (carrot-e) [inline to_hex.js]: `return returnValue;`?

BTW a better name for `returnValue` could be `output` or `result`?

File: sum_odd_numbs.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+let rowSumOddNumbers = n => {
```
Comment (carrot-e) [inline sum_odd_numbs.js]: Why not `const`?

Comment (dreyhiflden) [reply sum_odd_numbs.js]: Actually I think const need to immutability, but after your comment and this simple [article](https://mathiasbynens.be/notes/es6-const) I change my mind.

File: to_hex.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+      result = '';
+
+  while (value !== 0) {
+    remainder = value % 16;
+    value = parseInt((value / 16), 10);
+    result = hexVal[remainder] + result;
+  }
+
+  return `${result}`;
+
```
Comment (carrot-e) [inline to_hex.js]: You don't need an empty newline here.
Also, you don't need it on line 2.

Comment (carrot-e) [reply to_hex.js]: Why not simply `return result;`?

Comment (dreyhiflden) [reply to_hex.js]: Before this it was return ``0x${result}`;  ` and after delete prefix I forgot to change to just `return`

**Comment by dreyhiflden** [comment]
Okey, thanks

**Pull request merged**